test/test_helper.rb

```diff
@@ -1,7 +1,7 @@
-require "codeclimate-test-reporter"
+require 'codeclimate-test-reporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.